src/logger/MeetingSessionPOSTLogger.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Logger from '../logger/Logger';
+import LogLevel from '../logger/LogLevel';
+import IntervalScheduler from '../scheduler/IntervalScheduler';
+import MeetingSessionConfiguration from '../meetingsession/MeetingSessionConfiguration';
+
+export default class MeetingSessionPOSTLogger implements Logger {
```

Comment:
The name should be just POSTLogger. Following the comment I left below, logger should not be tied (or know) to a meeting session.